flink-tests/src/test/scala/org/apache/flink/api/scala/migration/ScalaSerializersMigrationTest.scala

```diff
@@ -28,11 +28,11 @@ import scala.util.Try
 class ScalaSerializersMigrationTest {
 
   /**
-   * Verifies that the generated classnames for anonymous Scala serializers remain the same.
-   *
-   * The classnames in this test are collected from running the same type information generation
-   * code in previous version branches. They should not change across different Flink versions.
-   */
+    * Verifies that the generated classnames for anonymous Scala serializers remain the same.
```

Comment:
It's hard to see what the actual changes are in this file because there are a lot of formatting changes.

Reply:
Yes, unfortunately most of them were needed to pass checkstyle.